public/javascripts/data/storyManager.js

```diff
@@ -0,0 +1,45 @@
+const Database = require('better-sqlite3');
+
+exports.displayAllStories = () => {
```

Comment:
You might consider accepting a limit parameter here in `displayAllStories` which would limit the number of story records returned by the DB. The reason being as your application gets more and more story submissions, the home page will continue to grow and become extremely long. When you want to render the homepage, you could pass a limit parameter of 5 or 10 and default to all stories if there's no limit parameter passed.

It's not a hard requirement, just a design consideration. This is the part of application development that begins to bleed over to more of the soft requirements side